### ISSUE TITLE
(dev 2.0) ofJson and toJson warnings

### ISSUE THREAD

**Issue by MangelMaxime** [fable-compiler/Fable#1483]
### Description

If we use `ofJson` or `toJson` we get this warnings:

```
This construct is deprecated. Currently not working for Fable 2, replaced by JSON.parse

This construct is deprecated. Currently not working for Fable 2, replaced by JSON.stringify
```

I think we should improve the message to mention Thoth.Json. Because, most of the time people will except it to convert F# types.

I could add a special page / section to Thoth documentation explanation the equivalent to `ofJson` and `toJson` in Thoth.Json v2

**Comment by alfonsogarciacaro** [comment]
True, I guess we can add a mention to Thoth.Json for now. When the documents are polished we can add a link, it should be also good to mention Fable.SimpleJson as well :+1:

**Comment by alfonsogarciacaro** [comment]
The warnings mention Thoth.Json son :)
